ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java

```diff
@@ -187,7 +188,7 @@ public void loadRewardedAd(
     VungleInitializer.getInstance()
         .initialize(
             appID,
-            context.getApplicationContext(),
+                mediationRewardedAdConfiguration.getContext().getApplicationContext(),
```

Comment:
what does this change do, is it related to the issue you saw earlier that AdMob had fixes